internal_transfers/actions/tests/orderbook.spec.ts

```diff
@@ -0,0 +1,43 @@
+    };
+
+    expect(await getSettlementCompeitionData(txHash)).toEqual(expected);
```

Comment:
You can use `.toMatchSnapshot()` to avoid storing of the huge data in the tests code

Reply:
Tell me more!

Reply:
Love this and did it. Thanks!